xdocs/usermanual/functions.xml

```diff
@@ -1499,6 +1502,36 @@ becomes:
 </properties>
 </component>
 
+
+<component index="&sect-num;.5.22" name="__timeShifting">
+<description>
+    <p>The timeShifting function returns a date in various formats with the specified amount of seconds/minutes/hours/days/months added</p>
```

Comment:
"in various formats" => "in configured format" or "in the given format" as the function will return one value, only.